client/extensions/woocommerce/components/widget-group/index.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+
+class WidgetGroup extends Component {
+	static propTypes = {
+		children: PropTypes.oneOfType( [
+			PropTypes.arrayOf( PropTypes.node ),
+			PropTypes.node
+		] ),
+		className: PropTypes.string,
+		maxColumns: PropTypes.number,
+		title: PropTypes.string,
+	}
+
+	renderRow = ( rowIndex, children ) => {
+		return (
+			<div className="widget-group__row-container" key={ rowIndex }>
+				{ children.map( ( child, index ) => {
+					return React.cloneElement( child, {
+						className: classNames( child.props.className, 'widget-group__row-item', 'card' ),
+						key: index,
+					} );
+				} ) }
+			</div>
+		);
+	}
+
+	renderChildren = () => {
```

Comment:
Fun! Took me a second to figure out why it was done this way -- so a comment saying how this will split children into correct rows with the correct number per would probably be helpful for future devs that might look in here.